classes/author/maps/Schema.inc.php

```diff
@@ -76,7 +91,21 @@ protected function mapByProperties(array $props, Author $item): array
     {
         $output = [];
         foreach ($props as $prop) {
-            $output[$prop] = $item->getData($prop);
+            switch ($prop) {
+                case 'userGroupLabel':
+                    $userGroupId = $item->getData('userGroupId');
+                    $output[$prop] = isset($this->userGroupLabels[$userGroupId])
+                        ? $this->userGroupLabels[$userGroupId]
+                        : '';
```

Comment:
Since this is called `name` in the `UserGroup` object, let's call this prop `userGroupName`. Also,  let's return `$userGroup->getName()` instead of `$userGroup->getLocalizedName()` so that a consumer of the API has all locales if they want them.

Reply:
Changed